adback-solution-to-adblock/admin/class-ad-back-admin.php

```diff
@@ -307,6 +307,10 @@ public function displayPluginDiagnosticPage()
             $script = $this->askScripts();
             $table_name_end_point = $wpdb->prefix . 'adback_end_point';
             $endPoints = $wpdb->get_row("SELECT * FROM " . $table_name_end_point . " WHERE id = ".get_current_blog_id());
+
+            $rules = get_option('rewrite_rules');
```

Comment:
you could use `$rules = get_option('rewrite_rules', array())`.

To also be compatible with php < 5.4